### DISCUSSION TITLE
Review 1.17

### DISCUSSION THREAD

**Discussion by ludwich66** [webspiderteam/Bluetooth-DMM-For-Windows#22] [General]
where is a good place to store the iconset?
[BT-DMM-ICONSET.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/9270578/BT-DMM-ICONSET.zip)
![image](https://user-images.githubusercontent.com/12202733/183128864-61bf4e8b-ceff-4aca-a137-138bd8aaf0e4.png)

_Originally posted by @ludwich66 in https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/discussions/17#discussioncomment-3335616_

**Comment by webspiderteam** [comment]
O already uploaded in extra folder

**Comment by ludwich66** [comment]
> O already uploaded in extra folder

now i seen it ;-)

**Comment by ludwich66** [comment]
Hi Engin,
you made a Dummy-View with Icons from BT-DMM-Iconset.
I don't find the Picture now - Can you tell me where i can it found?

L

**Reply by webspiderteam** [reply]
Just hold ctrl pressed and click menu and then settings without releasing ctrl

**Reply by ludwich66** [reply]
oh a nice feature ;-)

**Comment by ludwich66** [comment]
117DevVersion

An old problem?
Last measurement is "hanging" in QMTT 
Did you this not fixed before?

Mode = Auto 

<img width="311" alt="image" src="https://user-images.githubusercontent.com/12202733/184130518-cfd6afa6-fbc0-4374-8d71-2641bb820421.png">

<img width="1117" alt="image" src="https://user-images.githubusercontent.com/12202733/184130671-d4805ad3-4284-4a1b-b1c5-dabb000be060.png">


Otherwise it looks very good!
L

**Reply by webspiderteam** [reply]
[BluetoothDMM.zip](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/files/9308761/BluetoothDMM.zip)
that should work

**Comment by webspiderteam** [comment]
I was fixed but I forgot it in mqtt

**Reply by ludwich66** [reply]
;-) please send next test version or jump to 1.18 if it fixed

**Reply by webspiderteam** [reply]
Already sent https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/discussions/22#discussioncomment-3374328

**Comment by webspiderteam** [comment]
I will upload when your test is ok

**Reply by ludwich66** [reply]
Hello, I haven't noticed anything else.

You could add a small improvement to the labels in the CSV file: "Time" becomes "PeriodInSec"
General: Why don't you also use the timestamp that you built especially for MQTT?

I say GO !

ludwich
